internal/view/live_view.go

```diff
@@ -343,16 +341,7 @@ func (v *LiveView) saveCmd(evt *tcell.EventKey) *tcell.EventKey {
 }
 
 func (v *LiveView) cpCmd(evt *tcell.EventKey) *tcell.EventKey {
-	v.app.Flash().Info("Content copied to clipboard...")
-	if err := clipboard.Init(); err != nil {
-		v.app.Flash().Err(err)
-	} else {
-		if clipboard.Write(clipboard.FmtText, []byte(v.text.GetText(true))) == nil {
-			v.app.Flash().Err(errors.New("Failed to write to clipboard"))
-		}
-	}
-
-	return nil
+	return cpCmd(v.app.Flash(), v.text.GetText(true))(evt)
```

Comment:
No need to wrap here just make the call to cpCmd helper directly and axe LiveView.cpCmd.